applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -55,20 +65,21 @@ struct le_audio_headset {
 	atomic_t iso_tx_pool_alloc;
 	struct k_work_delayable stream_start_sink_work;
 	struct k_work_delayable stream_start_source_work;
+	bool qos_reconfigure;
 };
 
-struct worker_data {
-	uint8_t channel_index;
-	enum bt_audio_dir dir;
-	uint8_t retries;
-} __aligned(4);
-
 struct temp_codec_cap_storage {
 	struct bt_conn *conn;
 	/* Must be the same size as sink_codec_cap and source_codec_cap */
 	struct bt_codec *cap[CONFIG_BT_AUDIO_UNICAST_CLIENT_PAC_COUNT];
 };
 
+struct worker_data {
```

Comment:
Should do a rebase, so that these changes are not included

Reply:
I guess this was moved in some rebase/merge-conflict. This is just nitpick, but unless there is a good reason for why it was moved, it should be moved back as to not include this bit of code in the PR

Reply:
Hasn't been moved back